src/lang/po/de/Prusa-Firmware-Buddy_de.po

```diff
@@ -909,7 +909,7 @@ msgstr "Warnung"
 
 #: src/gui/wizard/screen_wizard.cpp:146
 msgid "Welcome to the     \nOriginal Prusa MINI\nsetup wizard.      \nWould you like to  \ncontinue?           "
-msgstr "Willkommen beim     \nOriginal Prusa MINI\nSetup-Assistenten.      \nMöchten Sie  \nfortfahren?"
+msgstr "Willkommen beim     \nOriginal Prusa MINI\nEinrichtungsassistenten.      \nMöchten Sie  \nfortfahren?"
```

Comment:
`Einrichtungsassistenten.` exceeds the 20 char limit per line.
You can use
`"Willkommen beim     \nOriginal Prusa MINI \nEinrichtungs-       \nassistenten. Möchten\nSie fortfahren?"`

Which will have 5 lines and look like
```
Willkommen beim
Original Prusa MINI
Einrichtungs-
assistenten. Möchten
Sie fortfahren?
```

Reply:
Then it should be Setup-assistenten in both. Because a few lines above that it's also called Einrichtungsassistenten.

Reply:
Einrichtungsassistent is correct but as there is a chars per line limit of 20 I split the word as the German rules allow it in two lines.